front/src/modules/trainschedule/components/TimetableV2/consts.ts

```diff
@@ -0,0 +1,17 @@
+import type { InvalidReason } from './types';
+
+const specialCodeDictionary: { [key: string]: string } = {
```

Comment:
remove the export default
```suggestion
export const specialCodeDictionary: { [key: string]: string } = {
```

Reply:
Done